tests/unit/amo/components/TestSearchResult.js

```diff
@@ -350,7 +371,7 @@ describe(__filename, () => {
   // TODO: This can be removed once migration happens.
   // See: https://github.com/mozilla/addons-frontend/issues/5359
```

Comment:
/me pretends he didn't see this TODO...

Reply:
should I delete this test then?  Despite the TODO it's testing fallback functionality that is still present (it's tested in test_imageUtils.js though wouldn't be in the component tests)

Reply:
Maybe just delete the comment, the issue linked has been fixed

Reply:
I also pretended not to see that... I almost suggested removing it, but maybe this deserves a full issue to remove the fallback altogether if that's obsolete ?

Reply:
the _need_ for a fallback is obsolete (that theme size will always exist) but the code would be pretty much the same until mozilla/addons-server#16638 drops the additional preview sizes.  Although even then I suppose this could happen in the future?  (the API/server makes no guarantee there would only ever be one preview per theme and the size frontend wants would be in a particular position in the array).

Reply:
Yep, I think we should keep the test case as well as the fallback.

Reply:
https://github.com/mozilla/addons-frontend/pull/10125/commits/d088b7d0c5cf133c1ede93455acf040acaf3e050